examples/lalrpop-app/app.rs

```diff
@@ -0,0 +1,21 @@
+extern crate lalrpop_util;
+
+use std::env;
+use std::fs;
+
+mod json;
+
+fn main() {
+    let src = fs::read_to_string(env::args().nth(1).expect("Expected file argument"))
+        .expect("Failed to read file");
+
+    match json::ObjectParser::new().parse(&src) {
```

Comment:
Isn't the root item in json a Value instead of an Object?

Reply:
Oh, yep.  I didn't read the spec closely enough it seems.  Will fix.